server/controllers/user.controller.js

```diff
@@ -201,6 +201,57 @@ class UserController {
     return Response(res, 200, `The user role has been changed to ${newrole}.`, [result]);
   }
 
+  /**
+     * @description gets the profile of the a user
+     * @param {object} req
+     * @param {object} res
+     * @returns {object} User Profile Object
+     */
+  static async getProfileById(req, res) {
+    const { id } = req.params;
+    const userProfile = await User.findByPk(id);
+    // get followers, following, and articles
+    if (!userProfile) return Response(res, 404, 'No profile found for this user.');
+    const {
+      dataValues: {
+        firstname,
+        lastname,
+        username,
+        bio,
+        image,
+      }
+    } = userProfile;
+
+    const profileObject = {
+      firstname, lastname, username, bio, image,
+    };
+    return Response(res, 200, 'User profile found.', [profileObject]);
+  }
+
+  /**
+     * @description edits a user's profile
```

Comment:
Hey @TheDrizzyWay the UserController class seems to be getting all packed up. What do you think about giving it a class of its own?
It would also help us in the separation of concerns since all other methods have to do with authentication.

Reply:
Alright, will do that.